code/menu.css

```diff
@@ -0,0 +1,118 @@
+.hamburger-menu-container {
+    position: relative;
+    top: 0;
+    right: 0;
+    z-index: 1;
+}
+
+/* CHECKBOX TO MAKE NAV MENU VISIBLE */
+.hamburger-menu-checkbox {
+    position: absolute;
+    top: -15px;
+    right: -3px;
+    z-index: 2;
+    cursor: pointer;
+    width: 30px;
+    height: 30px;
+    opacity: 0;
+}
+
+.hamburger-menu-illustration {
+    position: absolute;
+    top: 0;
+    right: 0;
+    z-index: 1;
+    width: 30px;
+    height: 5px;
+    background: black;
+    border-radius: 5px;
+    transition: all 0.4s ease;
+}
+
+.hamburger-menu-illustration::before,
+.hamburger-menu-illustration::after {
+    content: '';
+    position: absolute;
+    z-index: 1;
+    width: 30px;
+    height: 5px;
+    background: black;
+    border-radius: 5px;
+}
+
+.hamburger-menu-illustration::before {
+   top: -10px;
+}
+
+.hamburger-menu-illustration::after {
+    top: 10px;
+}
+
+/* ROTATION WHEN CHECKBOX IS CHECKED */
+.hamburger-menu-checkbox:checked + .hamburger-menu-illustration {
+    transform: rotate(90deg);
+}
+
+.hamburger-menu-checkbox:checked ~ .hamburger-menu-nav {
+    visibility: visible;
+}
+
+.hamburger-menu-nav {
+    position: fixed;
+    top: 89.9px;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    visibility: hidden;
+    overflow: hidden;
+    display: flex;
+    justify-content: center;
+    background: rgba(14, 13, 13, 0.8);
+    z-index: 2;
+}
+
+.hamburger-menu-nav ul {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
```

Comment:
This hamburger looks so cool!! To get it completely centered, you can add `padding: 0;` (to get rid of the `ul`s default padding)
<img width="326" alt="Skärmavbild 2021-02-03 kl  13 28 29" src="https://user-images.githubusercontent.com/51529201/106747076-b74b8380-6623-11eb-91f2-e4e416a693b8.png">